### ISSUE TITLE
Typescript Types missing in the npm package

### ISSUE THREAD

**Issue by bpasham** [yWorks/jsPDF#27]
Seems like types are well created but doesn't seem to be included in the npm package. Including them will help a lot of folks who are struggling to use this package in Typescript projects.

Also Thank you for all the hard work you put into this project.

**Comment by HackbrettXXX** [comment]
You are right. We also should update the types first, since they were merged/copied from upstream and are missing some methods from this fork.

**Comment by Danebrouwer97** [comment]
How do I get and use the typings file? I see they exist, but if I try copy+paste the code over into an index.d.ts file, TS just complains that it is not a module.
```
File '.../jspdf-yworks/index.d.ts' is not a module.
```
Edit: Found the solution. 
Change:
```
declare module 'jspdf' {
    // ... Interfaces ...
    class jsPDF { ... }
    namespace jsPDF { }

    export = jsPDF;
}
```
To:
```
declare namespace jsPDF {
    // ... Interfaces ...
    class jsPDF { ... }
}
export = jsPDF;
```
Update: 
This solves any issues thrown by typescript, but fails in actual use.  Likely because it's no longer a module that exports an actual class. I know nothing of creating Typings files, and only have a basic knowledge of producing typesafed code, so this is honestly all trial and error.
```
TypeError: jspdf_yworks__WEBPACK_IMPORTED_MODULE_5__.jsPDF is not a constructor
```

**Comment by HackbrettXXX** [comment]
Great you found the solution. A release where the types are included in the npm package is already in the pipeline.
